build/test-startupscriptgenerators.sh

```diff
@@ -4,20 +4,27 @@
 # Licensed under the MIT license.
 # --------------------------------------------------------------------------------------------
 
-set -ex
+set -e
 
 declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
-declare -r GEN_DIR="$REPO_DIR/src/startupscriptgenerator/"
+declare -r GEN_DIR="$REPO_DIR/src/startupscriptgenerator/src"
+
 # When volume mounting a directory from the host machine, we host it as a readonly folder because any modifications by a
 # container in that folder would be owned by 'root' user(as containers run as 'root' by default). Since CI build agents
 # run as non-root cleaning these files would be a problem. So we copy the mounted directory in the container
 # to a different directory within the container itself and run tests on it.
-declare -r GEN_DIR_CONTAINER_RO="/src"
-declare -r GEN_DIR_CONTAINER="/go/src/startupscriptgenerator"
-declare -r MODULE_TO_TEST="startupscriptgenerator/..."
+declare -r GEN_DIR_CONTAINER_RO="/startupscriptgenerator"
+declare -r GEN_DIR_CONTAINER="/go/src"
+declare -r MODULE_TO_TEST="..."
 declare -r CONTAINER_NAME="oryxtests_$RANDOM"
 
+runTests="echo && echo Running tests..."
+runTests="$runTests && cd $GEN_DIR_CONTAINER/common && go test -v "
+runTests="$runTests && cd $GEN_DIR_CONTAINER/dotnetcore && go test -v "
+runTests="$runTests && cd $GEN_DIR_CONTAINER/node && go test -v "
+runTests="$runTests && cd $GEN_DIR_CONTAINER/python && go test -v "
+runTests="$runTests && cd $GEN_DIR_CONTAINER/php && go test -v "
+
```

Comment:
I really do not like this but I am do not have another alternative. The thing is since _now_ on we are going to have folders like `github.com`, `gitlab.org` etc. I cannot run a `got test` on all files under `src` since we would be running tests under `github.com` too